pandas/tseries/tools.py

```diff
@@ -297,23 +297,22 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     NaT
 
     """
-    if origin == 'unix':
-        return _to_datetime(arg, errors=errors, dayfirst=dayfirst, yearfirst=yearfirst,
-                            utc=utc, box=box, format=format, exact=exact,
-                            unit=unit, infer_datetime_format=infer_datetime_format)
-    elif origin == 'julian':
+    offset = None
+    if origin == 'julian':
         if unit != 'D':
```

Comment:
The default for ``unit`` is ``ns``. So explicitly providing it as a ``None`` might be confusing (as julian dates are always defines as 'julian day'). Let me know what do you think, @jreback?